release-content/migration_guides_template.md

```diff
@@ -5,47 +5,8 @@ pull_requests: [14791, 15458, 15269]
 
 Copy the contents of this file into a new file in `./migration-guides`, update the metadata, and add migration guide content here.
 
-## Goals
-
-Aim to communicate:
+Remember, your aim to communicate:
 
 - What has changed since the last release?
 - Why did we make this breaking change?
 - How can users migrate their existing code?
```

Comment:
The content here was useful during writing, and I love links. Can we remind people to check the style guide in the instructions?

```suggestion
- How can users migrate their existing code?

For more specifics about style and content, see the [instructions](./migration_guides.md).
```